tests/integration-tests/tests/update/test_update.py

```diff
@@ -841,46 +842,44 @@ def _test_update_queue_strategy_with_running_job(
 def external_shared_storage_stack(request, test_datadir, region, vpc_stack: CfnVpcStack, cfn_stacks_factory):
     def create_stack(bucket_name):
         template_path = os.path.join(str(test_datadir), "storage-stack.yaml")
-        with open(template_path, encoding="utf-8") as template_file:
-            template = template_file.read()
```

Comment:
I moved the `with` statement down. You can use `Hide whitespace` to see more concise diff